[BUG] 因 WebKit 中的新 API `webkit_web_view_get_theme_color`，目前無法用 Debian trixie stable 套件編譯

**描述 Bug**

```
$ flutter build linux
github.com/sseu-buhzzi/DanXi/linux/flutter/ephemeral/.plugin_symlinks/flutter_inappwebview_linux/linux/in_app_webview/in_app_webview.cc:6214:23: error: use of undeclared identifier 'webkit_web_view_get_theme_color'                                                                                                                                                                                                             
Building Linux application...                                           
Build process failed
```

**复现步骤**
复现该 Bug 的步骤：
1. 當前 Debian trixie stable 中 WPEWebKit 仍然在 2.48.3-1：[libwpewebkit-2.0-dev/stable,now 2.48.3-1 amd64 [installed]](https://packages.debian.org/trixie/libwpewebkit-2.0-dev)
2. `webkit_web_view_get_theme_color` 自 2.50 後可用：https://github.com/WebKit/WebKit/blob/b8fb341d225d73ae242173579bba732fc2c1ca5d/Source/WebKit/UIProcess/API/wpe/WebKitWebViewWPE.cpp#L333-L359

**系统信息**
 - 系统版本：Debian GNU/Linux 13 (trixie)
 - 旦挞校园助手版本（见「设置-关于本应用」）：1.5.0+347

--- a/pubspec.yaml
+++ b/pubspec.yaml
@@ -125,6 +125,13 @@ dependency_overrides:
       ref: patch-2
       path: flutter_secure_storage_linux
 
+  # fixme: This fixes #645. This can be removed after the master branch fixes it.
+  flutter_inappwebview_linux:
+    git:
+      url: https://github.com/sseu-buhzzi/flutter_inappwebview.git
+      ref: 20260215/fix-webkit-check-version
+      path: flutter_inappwebview_linux
+
 dev_dependencies:
   build_runner: ^2.1.2
   pubspec_generator: ^5.0.0

